### ISSUE TITLE
DetailsList: aria-colindex not allowed within div

### ISSUE THREAD

**Issue by espeo196** [OfficeDev/office-ui-fabric-react#11259]
### Environment Information

- **Package version(s)**: Fabric 7
- **Browser and OS versions**: Chrome

### Describe the issue:

[HTML Validator](https://validator.w3.org/nu/about.html) for DetailsList are failing due to invalid area-colindex on divs. This validator is run as part of a Accessibility Insights for Web assessment. Specifically step 13: Parsing.

### Documentation describing expected behavior

Go to [DetailsList ](https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist) documentation page. To run the HTML Checker copy the link for "Check Serialized DOM of current page" and paste the copied script into the console for the DetailsList page.

Snippet from report:
![image](https://user-images.githubusercontent.com/4120503/69274203-b9e0cb00-0b8e-11ea-81c2-d24ed1621381.png)

**Comment by jdhuntington** [comment]
Hi @espeo196 - Is there a chance the validator is incorrect? Looking at the spec for `aria-colindex`, it looks like usage within a `div` is an expected case.

https://www.w3.org/TR/wai-aria-1.1/#aria-colindex

**Comment by espeo196** [comment]
This is a good point. Looking through the documentation it looks like aria-colindex is only required if the column is hidden from the DOM. Otherwise it is calculated automatically. See the entry here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Cell_Role. It is possible that the validator should be marking this as a warning rather than an error because the documentation doesn't say explicitly that it's not allowed.

**Comment by aneeshack4** [comment]
@espeo196 It sounds like this is an issue with the validator then so I'll go ahead and close this. Please don't hesitate to ping us if something else comes up.
